arduino/main/main.ino

```diff
@@ -52,10 +63,10 @@ LiquidCrystal_I2C lcd(0x38, 16, 2);
 TheThingsNetwork ttn(loraSerial, debugSerial, freqPlan);
 
 // AppEUI and AppKey for The Things Network
-const char *appEui = "70B3D57ED000F6F6";
-const char *appKey = "ECDEFF8CAAA80B6A7EF0522B027DEFFC";
+const char *appEui = "70B3D57ED001712B";
```

Comment:
We need to hide this (otherwise any Arduino can have access to the DB

Reply:
Sounds good

Reply:
Committed a change to hide the app key, kept app EUI since it is same across all devices